### PR TITLE
Fix/revpi 1016/mv svn GitHub

### PR DESCRIPTION
**Pull request by zhan-kunbus**
1. change the version to 2.0.0(no debian release number)
2. change the copyright
3. switch to native format

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -1,5 +1,38 @@
-Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: revpi-repo
```
Comment (iluminat23) [inline debian/copyright:2]: If there is no public repo, don't add those fields?

File: debian/copyright [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+Upstream-Contact: <preferred name and address to reach the upstream project>
+Source: <url://example.com>
 
 Files: *
-Copyright: 2016-2017 Kunbus GmbH
-License: GPL-2.0
+Copyright: 2021 Frank Pavlic <f.pavlic@kunbus.com>
```
Comment (iluminat23) [inline debian/copyright]: How is it that Frank is now the copyright holder? And why has the Kunbus GmbH lost its copyright from 2016?

**Review by iluminat23 — CHANGES_REQUESTED** [review]

File: debian/copyright (hunk 1 above)
Comment (zhan-kunbus) [reply debian/copyright:2]: Are you mean it is not (be able to) submitted to debian community?
btw, the revpi-repo is now (going to be) public on github

File: debian/copyright (hunk 2 above)
Comment (zhan-kunbus) [reply debian/copyright]: Oh, sorry, Frank is the Maintainer,  I made a mistake. So it should be still " 2016-2021 Kunbus GmbH" ?

File: debian/copyright (hunk 1 above)
Comment (iluminat23) [reply debian/copyright:2]: It looks ugly to have the template fields in the file. If it is public, add the repo. If not remove the lines!

File: debian/copyright (hunk 2 above)
Comment (iluminat23) [reply debian/copyright]: I guess so

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**